hs_core/views/utils.py

```diff
@@ -695,3 +716,11 @@ def move_or_rename_file_or_folder(user, res_id, src_path, tgt_path):
         rename_irods_file_or_folder_in_django(resource, src_full_path, tgt_full_path)
 
     hydroshare.utils.resource_modified(resource, user)
+
+
+def irods_path_is_allowed(path):
+    """ paths containing '/../' are suspicious """
+    if path == "":
+        raise ValidationError("Empty file paths are not allowed")
+    if '/../' in path:
+        raise SuspiciousFileOperation("File paths cannot contain '/../'")
```

Comment:
@hyi In any case, using /./ in a path should be meaningless. It is now flagged as an error.